vllm/config.py

```diff
@@ -151,6 +151,10 @@ def __init__(self,
                  use_async_output_proc: bool = True,
                  override_neuron_config: Optional[Dict[str, Any]] = None,
                  config_format: ConfigFormat = ConfigFormat.AUTO) -> None:
+
+        from vllm.plugins import load_general_plugins
+        load_general_plugins()
+
```

Comment:
can you move it to `EngineArgs.__post_init__(self)` ? it looks strange if we directly specialize it for model config.